### PR TITLE
Add extraEnv & remove duplicate label

### PR DESCRIPTION
**Pull request by covert8**
* Enables adding listen env var for ipv6 support fixing #2 

* Corrects duplicate label "helm.sh/charts" already present in "oci-registry.labels" causing kustomize builds to fail

### CODE REVIEW

**Comment by mcronce** [comment]
@covert8 thanks for the contribution!  This looks good to me - the only request I have is to undo the indentation changes in `deployment.yaml`, then I'll merge and cut a release

**Comment by covert8** [comment]
Thanks for working with me, you'll have to squash the pr but i hope its ok now?

File: dist/helm/templates/deployment.yaml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ template "oci-registry.fullname" . }}
+  name:  {{ template "oci-registry.fullname" . }}
   labels:
     {{- include "oci-registry.labels" . | nindent 4 }}
+    helm.sh/chart: {{ template "oci-registry.chart" . }}
```
Comment (mcronce) [inline dist/helm/templates/deployment.yaml]: You undid one of your changes :)

No problem though - when I get done my dayjob today I'll merge, fix this, and cut a release

Thanks for fixing it up!

Comment (covert8) [reply dist/helm/templates/deployment.yaml]: thanks for noticing, fixed now

**Comment by mcronce** [comment]
@covert8 this fix is included in the now-released v0.3.11.  Thanks again!

**Pull request merged**